src/code/HTTPRequestHandler/httprequesthandlerDELETE.cpp

```diff
@@ -0,0 +1,25 @@
+#include <HTTPRequestHandler/httprequesthandlerDELETE.hpp>
```

Comment:
the name of the dir should be the same as the files:

src/code/**HTTPRequestHandler**/**httprequesthandlerDELETE**.cpp
->
src/code/**httprequesthandlerDELETE**/**httprequesthandlerDELETE**.cpp